internal/doctor/model.go

```diff
@@ -0,0 +1,15 @@
+package doctor
+
+import (
+	"hospitalApp/internal/patient"
+	"time"
+)
+
+type Doctor struct {
+	DoctorId  uint              `json:"doctor_id" gorm:"primary_key; references"`
+	Name      string            `json:"name" gorm:"size:50"`
+	ContactNo string            `json:"contact_no" gorm:"size:10"`
```

Comment:
won't ensure 10 digits number. You can have the checks in request structs, then the model is okay

Reply:
okay, will correct this.